definitions/divorce/json/CaseEvent/CaseEvent-alt-service-more-case-forward-nonprod.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "LiveFrom": "30/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "serveByAlternativeMethod",
+    "Name": "Serve by alternative method",
+    "Description": "Serve by alternative method",
+    "DisplayOrder": "19",
+    "PreConditionState(s)": "GeneralConsiderationComplete, Issued",
```

Comment:
Needs to be separated with a ";" rather than ",".

Reply:
Momentary lapse! Thank you :D